README.rst

```diff
@@ -9,16 +9,16 @@ Contributing
 
 This project only requires that:
 
-#. you have `tox`_ installed
-#. you have Python 3 installed locally
+#. you have `sphinx`_ installed (`pip install sphinx`)
```

Comment:
Do we need a lower constraint? Travis will just grab the latest, in theory I suppose we could give a lower bound, but is it really needed?